openfermioncirq/gates/fermionic_simulation.py

```diff
@@ -133,9 +196,9 @@ def __init__(
         if absorb_exponent:
             self.absorb_exponent_into_weights()
 
-    @abc.abstractproperty
-    def fermion_generator_components(self
-                                    ) -> Tuple[openfermion.FermionOperator]:
+    @classmethod
```

Comment:
Use `@staticmethod` instead.

Reply:
Done. Do you see way of doing the same for `num_weights` without making it an `abstractproperty`?